build_extra/rust/rust.gni

```diff
@@ -49,6 +49,7 @@ template("rust_crate") {
                            "crate_type",
                            "crate_version",
                            "deps",
+                           "data",
```

Comment:
Not needed. We do not have any data deps for any rust targets.

Reply:
Removed.